packages/manual/VERSION

```diff
@@ -0,0 +1 @@
+/home/dev/manual/linux.d/versions/VERSION-5.10
```

Comment:
maybe use relative symbolic link, so it doesn't break if we ever change the build user or directory structure

Reply:
True, this should not be checked in. 

The symbolic link is created via the Makefile for each target, and uses `$(realpath manual)` to get the absolute path to the manual folder relativ to the makefile.
1. https://github.com/gardenlinux/gardenlinux/blob/18c9703be4c0bf9b9e2df0879fd429db3c797bf0/packages/Makefile#L2
2. https://github.com/gardenlinux/gardenlinux/blob/18c9703be4c0bf9b9e2df0879fd429db3c797bf0/packages/Makefile#L51

What do you think?